doc/source/flexibility/binlogging_replication_improvements.rst

```diff
@@ -108,10 +109,57 @@ condition. This fix has been ported from the upstream bug report
 :mysqlbug:`42415` (:psbug:`44`).
 
 Version Specific Information
-****************************
+--------------------------------------------------------------------------------
 
   * :rn:`5.6.13-60.5`
     Fix ported from |Percona Server| 5.5
 
+.. _percona-server.binary-log.flush.writing:
 
+Writing ``FLUSH`` Commands to the Binary Log 
+================================================================================
 
+``FLUSH`` commands, such as such as ``FLUSH SLOW LOGS``, are not written to the
```

Comment:
I would not combine the following 2 paragraphs

```
``FLUSH`` commands, such as such as ``FLUSH SLOW LOGS``, are not written to the
binary log if the system variable :variable:`binlog_skip_flush_commands` is set
to **ON**.

In addition, the following changes in the readonly / super_readonly behavior were implemented:
- When ``read_only`` is set to ON, any FLUSH XXX command executed by a normal user (w/o SUPER privilege) will not be written to the binary log regardless of the value of the |bsfc| variable.
- When ``super_read_only`` is set to ON, any FLUSH XXX command executed by any user (even by those with SUPER privilege) will not be written to the binary log regardless of the value of the |bsfc| variable.

An attempt to run a ``FLUSH`` command without either ``SUPER`` or ``RELOAD``
privileges, results in the ``ER_SPECIFIC_ACCESS_DENIED_ERROR`` exception
regardless of the value of the |bsfc| variable.
```

Reply:
Ready for review